src/chains/algorand_1/algoChainState.ts

```diff
@@ -380,7 +381,7 @@ export class AlgorandChainState {
       return
     }
     // not yet reached limit - set a timer to call this function again (in checkInterval ms)
-    const checkAgainInMs = checkInterval
+    // const checkAgainInMs = checkInterval
```

Comment:
Is this incomplete? - If so we dont want to check in WIP code into the final code base. Add a clear ToDo exampling what needs to be done and revert the WIP changes

Reply:
Also, open a clear ticket in Trello for what needs to be done here